.github/workflows/push.yml

```diff
@@ -7,6 +7,8 @@ on:
   push:
     branches-ignore:
       - 'master'
+      # any branch with a name ending in -no-build will not trigger this workflow.
+      - '*-no-build'
```

Comment:
Do I understand it correctly that this is needed to prevent the workflow from being run when you commit through the GH UI? If so, that seems to be a very fragile way of preventing the run as editor changes will actually prevent the runs unless we explicitly name the branch appropriately. At the very least, this trick should be documented properly in the README.

If I understood the use case properly, I personally think that there would not be any harm if the workflow is triggered on a documentation change (even though we don't need it to be run) and I would favour this over introducing "tricks" like this.

Reply:
> Do I understand it correctly that this is needed to prevent the workflow from being run when you commit through the GH UI? If so, that seems to be a very fragile way of preventing the run as editor changes will actually prevent the runs unless we explicitly name the branch appropriately. At the very least, this trick should be documented properly in the README.

This is sort of what it is.... a "hack" (or "developer tool") to cover the fact that:
```
 paths-ignore:
   - '*.md'
```
does not work as naively expected... It is not meant to prevent anything in general, just to open the opportunity of avoiding runs when explicitly desired. It is for when we make lots and lots of minor docs edits, via GitHub UI (maybe from a phone), and those edits generate a long list of non descriptive commits (that would we also probably later want to squash).

> If I understood the use case properly, I personally think that there would not be any harm if the workflow is triggered on a documentation change (even though we don't need it to be run) and I would favour this over introducing "tricks" like this.

I'm not strongly opinionated actually. The tradeoff is clear: multiple unneeded build runs vs. avoiding unneeded runs with "hackish" method. Votes? @cheempz 👍 👎 

(BTW - The [README](https://github.com/appoptics/appoptics-bindings-node/blob/AO-20192-Update-Documentation-no-build/.github/README.md) is editable and branch is marked `-no-build`. WIP. Edits welcomed... 😄 )

Reply:
I think it's fine to have the "hack" in, it's there for someone who is aware of it but it's harmless even if unaware to just trigger the unneeded runs.